packages/metrics-app/src/screens/HomeScreen.tsx

```diff
@@ -1,24 +1,78 @@
+import { useState } from 'react'
 import { StatusBar } from 'expo-status-bar'
-import { StyleSheet, Text, View } from 'react-native'
+import {
+  StyleSheet,
+  Text,
+  View,
+  TouchableOpacity,
+  // eslint-disable-next-line react-native/split-platform-components
+  ToastAndroid
+} from 'react-native'
+import AsyncStorage from '@react-native-async-storage/async-storage'
 import Tutorial from '../components/Tutorial'
-import { colors } from '../theme/colors'
+import { enableMessaging, disableMessaging } from '../services/messaging'
 
 
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    backgroundColor: colors.background.gray100,
     alignItems: 'center',
-    justifyContent: 'center'
+    justifyContent: 'center',
+    padding: 10
+  },
+  // eslint-disable-next-line react-native/no-color-literals
+  optinoutbutton: {
+    marginVertical: 10,
+    height: 40,
+    marginHorizontal: 10,
+    padding: 10,
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderRadius: 10,
+    fontSize: 16,
+    fontWeight: 'bold',
+    textTransform: 'uppercase',
+    borderWidth: 2,
+    borderColor: '#5d57ff',
+    textAlign: 'center',
+    textAlignVertical: 'center'
   }
 })
 
 function HomeScreen() {
+  const [ optIn, setOptIn ] = useState(true)
```

Comment:
When the app starts let's check the async storage, and by default set this state based on whether the user is actually opted in or out. A useEffect might be useful for this, and let's not render anything until we checked async storage - maybe an isLoading state could help? Also keep in mind that on first run the async storage is empty, in this case let's assume the user is opted in by default.